.travis.yml

```diff
@@ -161,6 +161,14 @@ env:
     env:
       - BEFORE_INSTALL="sed vm_opts.h -e 's/OPT_SUPPORT_JOKE *0/OPT_SUPPORT_JOKE 1/' -i"
 
+  - &CPDEBUG
+    name: CPDEBUG
+    <<: *gcc-8
+    <<: *cron-only
+    <<: *make-test-only
+    env:
+      - BEFORE_INSTALL="sed compile.c -e 's/define CPDEBUG 0/define CPDEBUG 1/' -i"
```

Comment:
You can pass a flag as `make DEFS=-DCPDEBUG`.

Reply:
As `DEFS` is kept unset in `Makefile`s so the environment variable precedes, you can set it here.

Reply:
Thanks! No need to bust out sed like `SUPPORT_JOKE` if a simple `-D` will do :-)